server_main.cc

```diff
@@ -66,7 +66,12 @@ int main(int argc, char* argv[]) {
   ParseArgs(argc, argv, parallelism, src_path, client_ip, timeout_msec);
   logger.reset(new SimpleLogger());
 
-  RpcChannel rpc(RpcChannel::Pier::Server, client_ip);
+  RepStatus rpc_rc;
```

Comment:
Why do define rpc_rc? We already have rc in this function.

Reply:
removed